### PR TITLE
docs: clearer support policy

### PR DESCRIPTION
**Pull request by shortcuts**
## Summary

Update the docs to make the support policy clearer to users.

### CODE REVIEW

File: packages/website/docs/DocSearch-program.md [hunk 1]
```diff
@@ -78,6 +78,18 @@ The information of the initially applied project is used everywhere when we depl
 
 This allow us to easily scope issues when reaching out for support.
 
+## Support
+
+We do not provide Algolia support for DocSearch, but for the products that are used by the program.
```
Comment (bodinsamuel) [inline packages/website/docs/DocSearch-program.md:83]: I'm not sure the sentence is clear. 
Maybe something like `We do not provide implementation support for DocSearch`

File: packages/website/docs/DocSearch-program.md [hunk 2]
```diff
@@ -78,6 +78,18 @@ The information of the initially applied project is used everywhere when we depl
...
+
+Please make sure to **first read the documentation (or [the Algolia one](https://www.algolia.com/doc/)) before reaching out** and take a look at [the Algolia academy](https://academy.algolia.com/trainings).
+
+For issues related to:
```
Comment (bodinsamuel) [inline packages/website/docs/DocSearch-program.md:87]: `bug report` might avoid confusion about what is an issue

File: packages/website/docs/DocSearch-program.md (hunk 1 above)
Comment (shortcuts) [reply packages/website/docs/DocSearch-program.md:83]: Make sense!

File: packages/website/docs/DocSearch-program.md (hunk 2 above)
Comment (shortcuts) [reply packages/website/docs/DocSearch-program.md:87]: I've removed this line with the suggestion you just made, it makes more sense

**Review by bodinsamuel — APPROVED** [review]

**Pull request merged**